gradle/javadoc.gradle

```diff
@@ -33,26 +33,25 @@ task docs(type: Javadoc) {
 
 def utilProject = project(':util')
 allprojects {
-    afterEvaluate { project ->
-        tasks.withType(Javadoc) {
-            dependsOn utilProject.compileJava //We need taglets to be compiled
-            exclude "**/com/mongodb/**/benchmark/**"
-            exclude "**/com/mongodb/**/internal/**"
-            exclude "**/org/bson/**/internal/**"
-            options {
-                author = true
-                version = true
-                links 'https://docs.oracle.com/en/java/javase/11/docs/api/'
-                tagletPath single(utilProject.sourceSets.main.output.classesDirs)
-                taglets 'ManualTaglet'
-                taglets 'DochubTaglet'
-                taglets 'ServerReleaseTaglet'
-                encoding = 'UTF-8'
-                charSet 'UTF-8'
-                docEncoding 'UTF-8'
-                addBooleanOption("html5", true)
-                addBooleanOption("-allow-script-in-comments", true)
-                header = '''
+    tasks.withType(Javadoc) {
```

Comment:
The reason I removed `afterEvaluate` is due to this warning:
```
Using method Project#afterEvaluate(Closure) when the project is already evaluated has been deprecated. This will fail with an error in Gradle 7.0. The configuration given is ignored because the project has already been evaluated. To apply this configuration, remove afterEvaluate.
```

Reply:
Is this actually ignored, as the warning suggests?  Do you see any difference in the generated Javadoc after making this change?

Reply:
sgtm